frontend/cypress/components/Transaction.cy.tsx

```diff
@@ -23,7 +23,7 @@ describe('<Transaction />', () => {
     });
 
     it('updates the transaction details after polling', () => {
-        cy.intercept('GET', `/api/transaction/${transactionData.userTransactionHash}`, {
+        cy.intercept('GET', `/api/transaction/hash=${transactionData.userTransactionHash}`, {
```

Comment:
Didn't we change it to `api/transaction/HASH` just in the previous PR?

Reply:
I had just removed the additional transaction keyword in the URL in the previous PR, but the correct API call is anyway without hash= so I've updated it now.